packages/tre/src/plugins/cache/index.ts

```diff
@@ -13,7 +14,12 @@ import {
 import { CacheGateway } from "./gateway";
 import { CacheRouter } from "./router";
 
-export const CacheOptionsSchema = z.object({
+export const CacheOptionsSchema = z.object({});
+export const CacheSharedOptionsSchema = z.object({
+  cachePersist: PersistenceSchema,
+  // Ideally, these options would be configurable per-worker (i.e. part of
+  // `CacheOptionsSchema` instead). However, `workerd` can only have one global
+  // "cache" service, so we can't distinguish which worker called the Cache API.
```

Comment:
I may be misremembering, but I’m pretty sure the cacheApiOutbound property in the capnp file that controls which Cache service to look at is per-worker, not global. Regardless, not worth supporting for right now